INFO     HTTP_LOGGER:event_hooks.py:27 Send POST request to http://localhost:8000/api/v1/users
INFO     HTTP_LOGGER:event_hooks.py:35 Got response: 200, reason: OK from http://localhost:8000/api/v1/users
INFO     SWAGGER_COVERAGE_TRACKER_STORAGE:storage.py:34 Results directory does not exist, creating: /home/runner/work/autotests-api/autotests-api/coverage-results
INFO     HTTP_LOGGER:event_hooks.py:27 Send POST request to http://localhost:8000/api/v1/authentication/login
INFO     HTTP_LOGGER:event_hooks.py:35 Got response: 200, reason: OK from http://localhost:8000/api/v1/authentication/login
INFO     BASE_ASSERTIONS:base.py:19 Check that response status code equals 200
INFO     AUTH_ASSERTIONS:authentication.py:18 Check login user response
INFO     BASE_ASSERTIONS:base.py:38 Check that "token_type" equals "bearer"
INFO     BASE_ASSERTIONS:base.py:56 Check that "access_token" is true
INFO     BASE_ASSERTIONS:base.py:56 Check that "refresh_token" is true
INFO     SCHEMA_ASSERTIONS:schema.py:20 Validate JSON schema